syncd/FlexCounter.cpp

```diff
@@ -114,6 +114,14 @@ FlexCounter::MACsecSAAttrIds::MACsecSAAttrIds(
     // empty intentionally
 }
 
+FlexCounter::TunnelCounterIds::TunnelCounterIds(
+        _In_ sai_object_id_t tunnel,
```

Comment:
tunnel vid or rid?

Reply:
tunnelRid. I modified variable name to reflect it.